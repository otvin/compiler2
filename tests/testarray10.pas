program testarray10(output);


var
	myarray:array[1..10] of real;
	i:integer;

begin

	myarray[1] := 1.1;
	myarray[2] := 1.1;
	for i := 3 to 10 do begin
		myarray[i] := myarray[i-1] + myarray[i-2];
	end;

	for i := 1 to 10 do begin
		writeln(myarray[i]);
	end;
end.
