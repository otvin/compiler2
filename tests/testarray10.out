1.100000000000
1.100000000000
2.200000000000
3.300000000000
5.500000000000
8.800000000000
14.300000000000
23.100000000000
37.400000000000
60.500000000000
